packages/usa-validation/src/index.js

```diff
@@ -3,7 +3,8 @@ const validate = require("../../uswds-core/src/js/utils/validate-input");
 const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
 const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
 
-const VALIDATE_INPUT = "input[data-validation-element]";
+const VALIDATE_INPUT =
+  "input[data-validation-element],textarea[data-validation-element]";
```

Comment:
## Validate input selector

Curious if there is a slightly cleaner way to define this selector. Probably not a blocker because this works well but may be room for a future enhancement in the future 🤔